modules/hosted-control-planes-troubleshooting.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-troubleshooting_{context}"]
+= Troubleshooting hosted control planes
+
+When you need to troubleshoot an issue with hosted control plane clusters, you can gather information by running the `hypershift dump cluster` command. The command generates output for the management cluster and the hosted cluster.
+
+The output for the management cluster contains the following content:
+
+* Cluster-scoped resources: These resources are node definitions of the management cluster.
+* The `hypershift-dump` compressed file: This file is useful if you need to share the content with other people.
+* Namespaced resources: These resources include all of the objects from the relevant namespaces, such as config maps, services, events, and logs.
+* Network logs: These logs include the OVN northbound and southbound databases and the status for each one.
+* HostedClusters: This level of output involves all of the resources inside of the hosted cluster.
```

Comment:
Is "HostedClusters" an actual thing? If it's a resource, probably should be marked up in backticks